mini-bots/DiscordCommands/SlashCommands.cs

```diff
@@ -111,6 +115,48 @@ private static string GetCommandUsage(string commandName, MethodInfo method)
             return usageBuilder.ToString().Trim();
         }
 
+        [SlashCommand("disable", "Disable a Minibot without deleting it.")]
+        public async Task DisableCommand(InteractionContext ctx, [Option("BotName", "who will it be")] string name)
+        {
+            MiniBot? miniBot = _databaseManager.GetMiniBotByName(name);
+
+            DiscordEmbedBuilder embedBuilder = new DiscordEmbedBuilder();
+
+            if (miniBot != null)
+            {
+                _databaseManager.SetMiniBotEnabled(miniBot.Id, false);
+                embedBuilder.WithTitle("Disabled bot: " + miniBot.Name);
+            }
+            else
+            {
+                embedBuilder.WithTitle("No bot with name: " + name);
+                embedBuilder.WithDescription("List bots: /list");
+            }
+
+            await Reponse(ctx, embedBuilder.Build());
+        }
+
+        [SlashCommand("enable", "Enable a Minibot.")]
+        public async Task EnableCommand(InteractionContext ctx, [Option("BotName", "who will it be")] string name)
```

Comment:
These two commands could be simplified by making a new function, as most of the code is the same

Reply:
fixed